### PR TITLE
Typo fixes + UI fixes

### PR DESCRIPTION
**Pull request by dominika-kokowicz**
Typo fixes + UI fixes

### CODE REVIEW

File: App/Utilities/Sources/Utilities/Design/Padding/DesignSystem.Padding.swift [hunk 1]
```diff
@@ -53,7 +53,7 @@ extension DesignSystem.Padding {
         switch self {
         case .tiny(let edges):          return edges
         case .smaller(let edges):       return edges
-        case .smal(let edges):          return edges
+        case .small(let edges):          return edges
```
Comment (sloik) [inline App/Utilities/Sources/Utilities/Design/Padding/DesignSystem.Padding.swift:56]: please align returns :D

File: App/Lettuce/Sources/Lettuce/UI/MessagesView/ThreadPreview.swift [hunk 2]
```diff
@@ -70,10 +67,12 @@ extension ThreadPreview: View {
 
                             }
                         }
-                        .padding(.trailing, 43)
+                        .padding(.trailing, 8)
```
Comment (sloik) [inline App/Lettuce/Sources/Lettuce/UI/MessagesView/ThreadPreview.swift:70]: please use `.design(padding:` that is closest :)

File: App/Lettuce/Sources/Lettuce/UI/MessagesView/ThreadPreview.swift [hunk 3]
```diff
@@ -23,6 +23,7 @@ extension ThreadPreview: View {
                 HStack(spacing: 0) {
 
                     InterlocutorAvatar(interlocutor: model.thread.interlocutor)
+                        .design(padding: .smaller(.vertical))
```
Comment (sloik) [inline App/Lettuce/Sources/Lettuce/UI/MessagesView/ThreadPreview.swift:26]: Love it <3

File: App/Lettuce/Sources/Lettuce/UI/MessagesView/Subviews/LastMessageView.swift [hunk 4]
```diff
@@ -4,17 +4,23 @@ import SwiftUI
 struct LastMessageView: View {
 
     let message: Message
+    let read: Bool
 
     private let fontColor = (Color(red: 0.24, green: 0.24, blue: 0.26).opacity(0.6))
+    private let readFontColor = (Color(red: 0.24, green: 0.24, blue: 0.26).opacity(1.0))
```
Comment (sloik) [inline App/Lettuce/Sources/Lettuce/UI/MessagesView/Subviews/LastMessageView.swift:10]: Please check `DesignSystem.Color`. You can add a custom type (if needed) for the message colors but I think we should move as much as possible to `DesignSystem`.

File: App/Lettuce/Sources/Lettuce/UI/MessagesDetailChatView/MessageDetailNavigationView.swift [hunk 5]
```diff
@@ -118,31 +113,33 @@ struct MessageDetailNavigationView: View {
...
             self.isInterlocutor = isInterlocutor
             self.alignment = isInterlocutor ? .leading : .trailing
             self.horizontalAlignment = isInterlocutor ? .leading : .trailing
+            self.messageText = messageText
         }
 
         var body: some View {
             VStack(alignment: horizontalAlignment, spacing: 0) {
-                Text("Hi! your last shot was realy good!")
+                Text(messageText)
```
Comment (sloik) [inline App/Lettuce/Sources/Lettuce/UI/MessagesDetailChatView/MessageDetailNavigationView.swift:133]: Hint: there is an extension that makes a `String` a view so in cases like those you can just write a string or use a string property. In other words `messageText` should be enough :)

**Review by sloik — APPROVED** [review]
Left some minor comments :)

File: App/Utilities/Sources/Utilities/Design/Padding/DesignSystem.Padding.swift (hunk 1 above)
Comment (dominika-kokowicz) [reply App/Utilities/Sources/Utilities/Design/Padding/DesignSystem.Padding.swift:56]: fixed

File: App/Lettuce/Sources/Lettuce/UI/MessagesView/ThreadPreview.swift (hunk 2 above)
Comment (dominika-kokowicz) [reply App/Lettuce/Sources/Lettuce/UI/MessagesView/ThreadPreview.swift]: fixed

File: App/Lettuce/Sources/Lettuce/UI/MessagesView/Subviews/LastMessageView.swift (hunk 4 above)
Comment (dominika-kokowicz) [reply App/Lettuce/Sources/Lettuce/UI/MessagesView/Subviews/LastMessageView.swift]: fixed

File: App/Lettuce/Sources/Lettuce/UI/MessagesDetailChatView/MessageDetailNavigationView.swift (hunk 5 above)
Comment (dominika-kokowicz) [reply App/Lettuce/Sources/Lettuce/UI/MessagesDetailChatView/MessageDetailNavigationView.swift]: fixed

**Pull request merged**